python/composio/tools/local/codeanalysis/actions/get_class_info.py

```diff
@@ -42,6 +43,7 @@ class GetClassInfo(
     def execute(
         self, request: GetClassInfoRequest, metadata: Dict
     ) -> GetClassInfoResponse:
+        CreateCodeMap().execute(CreateCodeMapRequest(), metadata)
```

Comment:
The addition of `CreateCodeMap().execute(CreateCodeMapRequest(), metadata)` in multiple places seems to introduce code duplication. Consider refactoring this into a shared method or utility if it's a common pattern that can be reused to keep the code DRY (Don't Repeat Yourself).